src/ServiceFabric/ServiceFabric/help/New-AzServiceFabricCluster.md

```diff
@@ -322,7 +322,7 @@ The Operating System of the VMs that make up the cluster.
 Type: Microsoft.Azure.Commands.ServiceFabric.Models.OperatingSystem
 Parameter Sets: ByDefaultArmTemplate
 Aliases: VmImage
-Accepted values: WindowsServer2012R2Datacenter, WindowsServer2016Datacenter, WindowsServer2016DatacenterwithContainers, UbuntuServer1604, UbuntuServer1804, UbuntuServer2004, WindowsServer2022
```

Comment:
this should not get removed

Reply:
I did not remove this manually. I think it got removed when I ran the update command on the help docs.

Reply:
Regardless you'll notice that the removed value is still a valid value https://github.com/Azure/azure-powershell/blob/main/src/ServiceFabric/ServiceFabric/Models/OperatingSystem.cs